chains/sapphiretestnet/metadata.yaml

```
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://nexus.oasis.io/v1/
+    family: other
+    name: Oasis Explorer
+    url: https://explorer.oasis.io/testnet/sapphire
+chainId: 23295
```

add deployer details
```suggestion
chainId: 23295
deployer:
  name: <project name>
  url: <project site>
```